### PR TITLE
More minor spelling fixes

### PR DESCRIPTION
**Pull request by BryanQuigley**
Forwarded 2 spelling fixes from Debian - and added 11 more while doing it.

### CODE REVIEW

File: src/cds_objects.cc [hunk 1]
```diff
@@ -255,7 +255,7 @@ void CdsItemInternalURL::validate()
     CdsItemExternalURL::validate();
 
     if (startswith(this->location, "http://"))
-        throw_std_runtime_error("Internal URL item validation failed: only realative URLs allowd");
+        throw_std_runtime_error("Internal URL item validation failed: only realative URLs allowed");
```
Comment (KarlStraussberger) [inline src/cds_objects.cc:258]: relative

**Review by KarlStraussberger — CHANGES_REQUESTED** [review]
Thanks. Found one more!

**Comment by BryanQuigley** [comment]
Added relative -ty. Do you want me to squash these all to one commit?

**Comment by KarlStraussberger** [comment]
Thank you

**Pull request merged**